Classes/People/PeopleViewController.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  PeopleViewController.swift
+//  Freetime
+//
+//  Created by Ryan Nystrom on 11/19/17.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import UIKit
+
+protocol PeopleViewControllerDelegate: class {
+    func didDismiss(controller: PeopleViewController, selections: [User])
+}
+
+final class PeopleViewController: UITableViewController {
+
+    enum PeopleType {
+        case assignee
+        case reviewer
+    }
+
+    private let selectionLimit = 10
+    private weak var delegate: PeopleViewControllerDelegate?
+    private var owner: String!
+    private var repo: String!
+    private var client: GithubClient!
+    private var users = [User]()
+    private let feedRefresh = FeedRefresh()
+    private var type: PeopleType!
+    private var selections = Set<String>()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        tableView.refreshControl = feedRefresh.refreshControl
+        feedRefresh.refreshControl.addTarget(self, action: #selector(LabelsViewController.onRefresh), for: .valueChanged)
```

Comment:
`LabelsViewController` here. Fun fact you don't need to specify the class `#selector(onRefresh)` is just as valid :)

Reply:
TIL